src/wwwroot/services/settings.js

```diff
@@ -231,5 +232,25 @@
         url: RELAY_CONFIG.cuitServiceBaseUrl + '/taxinfo/by-cuit/' + cuit
       });
     }
+
+    function resubscribeEmailAddress (subscriberEmail, reason, domainName, onExpectedError) {
+      var url = RELAY_CONFIG.baseUrl
+        + '/accounts/' + auth.getAccountName()
+        + '/subscribers/'
+        + subscriberEmail
+        + '/resubscribe';
+
+      var data = {};
+      data['reason'] = reason;
+      data['domain'] = domainName;
+
+      return $http({
+        actionDescription: 'action_requesting_resubscribe_email_address',
+        //tryHandleError: function(rejection){ return tryHandleErrorDomainManager(rejection, onExpectedError); },
```

Comment:
What is this commented code?

Reply:
I forgot to delete it, it is to handle errors, but it is for when we register new domains.

Reply:
What do you think about this (for the moment):

```suggestion
        // TODO: add tryHandleError to deal with expected errors
```